### ISSUE TITLE
[DataCap Removal Proposal] Abusive Behaviour Detected on 1924, 1920, 1919, 1918, 1929, 1923, 1922, 1921 & 1920

### ISSUE THREAD

**Issue by herrehesse** [filecoin-project/notary-governance#937]
## Issue Description

Nine instances of abusive and connected clients have been identified involving the following activities:
- Almost fully self-dealing their own applications
- Utilising VPNs to conceal their true location, indicating that miners are not operating from the claimed location. 
- These four separate clients belong to the same entity with funding wallet: f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
- These clients & wallets are linked to previous disputes on 1914, 1913, 1912, 1413, 1412, 1408 and 1404 [LINK](https://github.com/filecoin-project/notary-governance/issues/915).

This situation serves as a prime illustration where the client is also the storage provider, receiving support from negligent notaries who fail to fulfil their responsibilities.

## Impact

This organisation has the capability to accumulate datacap exclusively for its own miners, disregarding regional distribution and engaging in significant self-dealing. Such actions contradict the rules and guidelines of the Filecoin+ program, granting them undue financial advantages over entities striving to operate fair and decentralised storage provider businesses.

## Details

**Clients**

ID | Link | Title | Client | Funding Wallet
-- | -- | -- | -- | --
1918 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1918 | <Agricultural Home> - <Agricultural Home(1/3)> | [xiaoming123222](https://github.com/xiaoming123222) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1919 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1919 | <Agricultural Home> - <Agricultural Home(2/3) | [xiaoming123222](https://github.com/xiaoming123222) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1920 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1920 | <Agricultural Home> - <Agricultural Home(3/3) | [xiaoming123222](https://github.com/xiaoming123222) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1921 |  https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1921 | <Organization> - <Project Name> | [zhong-ta](https://github.com/zhong-ta) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1922 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1922 | <PhotographicFeeling> - <Photographic> | [zhong-ta](https://github.com/zhong-ta) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1923 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1923 | <Organization> - <Project Name> | [zhogntao](https://github.com/zhogntao) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1924 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1924 | <Agricultural Home> - <Agricultural Home(1/2)> | [zhogntao](https://github.com/zhogntao) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
1929 | https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1929 | <Beijing Bazuo Architectural Design Firm> - <Architectural Design（1/2）> | [zhogntao](https://github.com/zhogntao) | f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa

![Screenshot 2023-07-20 at 15 18 50](https://github.com/filecoin-project/notary-governance/assets/70132857/1b138e52-a5f4-495d-873d-3e77ace2898f)
![Screenshot 2023-07-20 at 15 19 00](https://github.com/filecoin-project/notary-governance/assets/70132857/e35af317-2bf5-4b2f-9488-81f7b09be8b1)

![Screenshot 2023-07-20 at 15 19 06](https://github.com/filecoin-project/notary-governance/assets/70132857/b127b7b7-0b43-499d-8b25-118ecc3ef90f)

![Screenshot 2023-07-20 at 15 19 12](https://github.com/filecoin-project/notary-governance/assets/70132857/63f2048c-8887-49e5-914a-0aaacc016c87)
![Screenshot 2023-07-20 at 15 19 20](https://github.com/filecoin-project/notary-governance/assets/70132857/1f45ac35-6088-43cc-a45a-92016eb1ff5b)

**Storage Providers**

Storage Provider | Client | IP Address | Port | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | -- | --
f01859603 | f02128499 | 14.18.234.122 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 13
f01923786 | f02128499 | 14.215.165.37 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 7
f01923787 | f02128499 | 14.215.165.40 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | TRUE with fraudscore 82 | 6
f01927554 | f02128499 | 14.215.165.46 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 16
f01938671 | f02128499 | 103.201.24.42 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 12
f01938674 | f02128499 | 183.60.90.194 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 15
f01836766 | f02128499 | 14.215.165.41 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | TRUE with fraudscore 82 | 10
f02237006 | f02128499 | 91.205.228.128 | 56801 | zoen | GB , England , Whitley Bay | TRUE with fraudscore 75 | 9
f02237010 | f02128499 | 23.224.22.163 | 56802 | ipfsfy.com | US , California , Los Angeles | TRUE with fraudscore 94 | 8
f01877184 | f02128499 | 8.219.103.232 | 50001 | Customers of Mike | SG , Singapore , Singapore | TRUE with fraudscore 75 | 4

Storage Provider | Client | IP Address | Port | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | -- | --
f01859603 | f02128612 | 14.18.234.122 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 15
f01923786 | f02128612 | 14.215.165.37 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 6
f01923787 | f02128612 | 14.215.165.40 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | TRUE with fraudscore 82 | 5
f01927554 | f02128612 | 14.215.165.46 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 13
f01938671 | f02128612 | 103.201.24.42 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 15
f01938674 | f02128612 | 183.60.90.194 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 11
f02237006 | f02128612 | 91.205.228.128 | 56801 | zoen | GB , England , Whitley Bay | TRUE with fraudscore 75 | 9
f01836766 | f02128612 | 14.215.165.41 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | TRUE with fraudscore 82 | 9
f02237010 | f02128612 | 23.224.22.163 | 56802 | ipfsfy.com | US , California , Los Angeles | TRUE with fraudscore 94 | 9
f01877184 | f02128612 | 8.219.103.232 | 50001 | Customers of Mike | SG , Singapore , Singapore | TRUE with fraudscore 75 | 0
f02252173 | f02128612 | 154.93.3.224 | 21153 |   | US , California , San Jose | TRUE with fraudscore 75 | 8

Storage Provider | Client | IP Address | Port | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | -- | --
f01859603 | f02128613 | 14.18.234.122 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 19
f01923787 | f02128613 | 14.215.165.40 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | TRUE with fraudscore 82 | 8
f01938671 | f02128613 | 103.201.24.42 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 18
f01923786 | f02128613 | 14.215.165.37 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 6
f01927554 | f02128613 | 14.215.165.46 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 14
f01938674 | f02128613 | 183.60.90.194 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 17
f02237006 | f02128613 | 91.205.228.128 | 56801 | zoen | GB , England , Whitley Bay | TRUE with fraudscore 75 | 9
f02237010 | f02128613 | 23.224.22.163 | 56802 | ipfsfy.com | US , California , Los Angeles | TRUE with fraudscore 94 | 7
f02252173 | f02128613 | 154.93.3.224 | 21153 |   | US , California , San Jose | TRUE with fraudscore 75 | 1

Storage Provider | Client | IP Address | Port | Name | Location | VPNresults | % Sealed
-- | -- | -- | -- | -- | -- | -- | --
f01923786 | f02129072 | 14.215.165.37 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 23
f01859603 | f02129072 | 14.18.234.122 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 13
f01923787 | f02129072 | 14.215.165.40 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | TRUE with fraudscore 82 | 19
f01927554 | f02129072 | 14.215.165.46 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 12
f01938671 | f02129072 | 103.201.24.42 | 50001 | Customers of Mike | HK , Hong Kong , Hong Kong | TRUE with fraudscore 80 | 15
f01938674 | f02129072 | 183.60.90.194 | 50001 | Customers of Mike | CN , Guangdong , Guangzhou | FALSE with fraudscore 0 | 17
f02237010 | f02129072 | 23.224.22.163 | 56802 | ipfsfy.com | US , California , Los Angeles | TRUE with fraudscore 94 | 0

**Notaries**

11ane-1
1igoovo
1ipollo00
1Normalnoise
1woshidama323

11ane-1
1igoovo
1ipollo00
1Normalnoise

11ane-1
1igoovo
1ipollo00
1Normalnoise

1igoovo
1zcfil

11ane-1
1DaYouGroup
1igoovo
1mikezli
1Normalnoise

(RECURRING NOTARIES INVOLVED IN ABUSIVE SIGNING)

## Proposed Solution(s)

We propose the following actions to address the situation:
- Immediately eliminate the remaining datacap granted on clients.
- Initiate disputes against notaries who have signed inappropriately.
- Immediate removal of repeated offending notaries from the multisig.
- Blacklist the storage providers implicated in the incident.

### Timeline

Discussion at July 25th 2023 WG Call
Community open audit until July 27th 2023 at 5 pm ET
Decision announced async at 5:01 pm ET

**Comment by xiaoming123222** [comment]
All CID reports are normal. Why do we say we abused them？

**Comment by xiaoming123222** [comment]
I have already explained the relevant issues clearly in #1918. Do you have any further questions?

**Comment by c12xf** [comment]
@herrehesse  The reason why I did not provide a reasonable explanation for the previous issue of #914  is because someone else provided evidence to your team on #913 proving that you also have the 'same entity', but you did not provide any explanation. I think you're making trouble for no reason

**Comment by dahuajishu** [comment]
@herrehesse Please ask your team to provide a reasonable explanation

**Comment by dahuajishu** [comment]
I found that most of our cid reports are allocated normally and reasonably, so what is abuse

**Comment by herrehesse** [comment]
I find it truly astonishing to read the content above. Are all three of us observing the same evidence presented here? Let me emphasize the following point:

- In any discussion, evidence can only be countered by presenting counter-evidence. Mere words hold little value as they are easily spoken. If you wish to challenge the valid and well-supported statements we have made, then please back your counterarguments with evidence.

I believe that any dialogue should be grounded solely in factual information, without any room for subjective opinions or personal approvals concerning our disputes. Let's focus on the facts and evidence at hand, leaving aside any need for validation or personal judgments.

**Comment by xiaoming123222** [comment]
What you call evidence of abuse is the same wallet address, the same miners?

**Comment by herrehesse** [comment]
@xiaoming123222 Can you see this below and tell me there is nothing wrong?

![image](https://github.com/filecoin-project/notary-governance/assets/70132857/16f1b963-3f3c-4549-97a1-3e724c29ac3e)

**Comment by xiaoming123222** [comment]
![image](https://github.com/filecoin-project/notary-governance/assets/130982779/ecb35df7-6cd4-47e2-bc33-93881a3429a9)
This is the latest one I have checked. Is there any problem?

**Comment by herrehesse** [comment]
Choosing to be ignorant is entirely up to you, and it seems like you have made that choice. As we have repeatedly pointed out, the distribution has been manipulated through VPN usage, and the evidence supporting this claim is undeniable. There's no need to further debate this matter.

Wishing you a pleasant day, my friend.

**Comment by xiaoming123222** [comment]
How could this become ignorance? This is my latest CID report from 1918, 1919, and 1920. Do you have any doubts about the CID report? Regarding the issue with VPN, SP informed me that they did not use VPN. Based on your inspection results, the credibility of the VPN inspection website you used is not high. Isn't it?

**Comment by herrehesse** [comment]
- In any discussion, evidence can only be countered by presenting counter-evidence. Mere words hold little value as they are easily spoken. If you wish to challenge the valid and well-supported statements we have made, then please back your counterarguments with evidence.

**Comment by xiaoming123222** [comment]
Are you making trouble here for no reason? Aren't these evidence?
https://github.com/data-preservation-programs/filplus-checker-assets/blob/main/filecoin-project/filecoin-plus-large-datasets/issues/1920/1689925056519.md

**Comment by herrehesse** [comment]
@xiaoming123222 

You really have no idea what the rules are do you? Your "proof" IS abuse.
Please take another good read of the rules here my friend: https://github.com/filecoin-project/filecoin-plus-large-datasets

**Comment by cryptowhizzard** [comment]
Nine instances of abusive and connected clients have been identified involving the following activities:

- Almost fully self-dealing their own applications
- Utilising VPNs to conceal their true location, indicating that miners are not operating from the claimed location.
- These four separate clients belong to the same entity with funding wallet: f1oamg2yffhxeaoc4gqm4tq6c4qowt3yn666loowa
- These clients & wallets are linked to previous disputes on 1914, 1913, 1912, 1413, 1412, 1408 and 1404 https://github.com/filecoin-project/notary-governance/issues/915.

**Comment by xiaoming123222** [comment]
- Isn't the LDN I applied for myself assigned by myself, such as generating cars and sending deals
-  they have clearly informed me that they are not using VPN
- Does the same wallet prove to be the same entity? It doesn't make sense, does it
- These numbers apply to companies from all over the world in China. Are they the same boss?

**Comment by xiaoming123222** [comment]
You're completely ignoring my evidence, aren't you?

**Comment by xiaoming123222** [comment]
Your so-called 'evidence' is not convincing. Please check the companies associated with each LDN to see if they are related. Just one wallet address can bind these dozens of companies together. Do you think this is a problem?

**Comment by herrehesse** [comment]
> Your so-called 'evidence' is not convincing. 

You are not the one deciding here my friend.

**Comment by xiaoming123222** [comment]
If you think your 'evidence' is convincing enough, then you can explain why #913 was closed by official personnel.

**Comment by herrehesse** [comment]
That is a mistake, we already discussed with FIL+ admins.

**Comment by xiaoming123222** [comment]
![image](https://github.com/filecoin-project/notary-governance/assets/130982779/d496a193-e7e4-4ce6-b375-1caf0ae467b5)
Why is your answer so funny when you are suspected to be the same entity

Why is it closed? Is it because your operation does not comply with the rules, and you feel guilty?
![image](https://github.com/filecoin-project/notary-governance/assets/130982779/e4f02b3d-a386-4a1f-a27e-98b5793dc3eb)

**Comment by herrehesse** [comment]
All of these have been answered a thousand times. Employing a defensive strategy and resorting to gaslighting are common tactics used by entities like yours, but we won't engage in such practices.

Wishing you success, my friend.

**Comment by xiaoming123222** [comment]
Wishing you success, my friend.

**Comment by igoovo** [comment]
There are few notaries active in the fil network.
It is inevitable to sign in pairs.

**Comment by herrehesse** [comment]
@igoovo Any words on the above evidence?

**Comment by igoovo** [comment]
I was so angry by your remarks that I didn't want to talk.
My friend, you might as well directly advise the authorities to "cancel all notaries in Greater China".
Only your honorable European notary. OK?

**Comment by herrehesse** [comment]
@igoovo, we're sorry to hear that you're feeling upset. Our main goal is to promote a healthier environment within the Filecoin+ program by exposing entities that are not following the rules. If you find yourself entangled in such a situation and have evidence to support your claims, the best course of action is to present that evidence. Emotions and words can often complicate matters.

If you believe there's evidence against accusations of self-dealing, incorrect data, or lack of due diligence, we encourage you to share your response with supporting evidence. However, please understand that in this thread, we prioritize evidence-based discussions over personal opinions.

We hope you can find some calm and still enjoy your weekend.

**Comment by c12xf** [comment]
Your team's racial discrimination is very serious, this is not the first time

**Comment by herrehesse** [comment]
Statements like "_Your team's racial discrimination is very serious_" may arise as a last resort in a discussion when there is no substantial evidence against the accused. 

It is important to clarify that observing abuse cases originating from a specific region, such as Asia/China, does not equate to racism. The focus is on addressing the behaviour, not discriminating against Chinese or Asians at all.

Understanding this distinction is crucial in having a fair and constructive conversation.

I encourage you once more to participate in the conversation with counter evidence, as opinions alone are not considered valid evidence. Your input based on factual information would be highly valued.

**Comment by c12xf** [comment]
I have checked several new LDNs applied for in China, and you have tried your best to prevent each one, which is affecting the judgment of official personnel

**Comment by herrehesse** [comment]
@c12xf Almost all applications are from CN, that is not racism. Also, read the rules:

- If a client wants to onboard more than 5+PiBs, the recommendation would be to start with a few applications and earn trust from the community. Having a positive reputation and proving to the community first by onboarding a smaller amount of data will help anyone who wants to onboard massive amounts of data much faster and smoother.

**Comment by c12xf** [comment]
As far as I know, the places where LDN is most commonly used come from two countries, the United States and China. Why didn't you investigate the LDN in the United States.

**Comment by c12xf** [comment]
Once again, the so-called 'evidence' you provided is difficult to convince

**Comment by herrehesse** [comment]
@c12xf, I recommend that you review the first 200 applications and check how many are from the USA. I can confidently say it's less than 5%.

If you possess counter evidence, please present it. Otherwise, let's avoid arguments based on feelings.

**Comment by c12xf** [comment]
I'll go find it

**Comment by c12xf** [comment]
@herrehesse Please review this [link](https://filplus.starboard.ventures/)

**Comment by herrehesse** [comment]
@c12xf So, after spending a full 24 hours manually reviewing all the applications, you discovered that over 95% of them originated from individuals of Asian descent, primarily Asian miners or related to Asian data? However, this information did not support the claim you were trying to make. As a result, you decided to share a link to the FIL+ website, which tracks miner activity and datacap usage, regardless of VPN use or IP location. Which has nothing to do with LDNs.

Let me rephrase your request once more: **Please assess the first 200 applications and determine how many of them come (organically) from the United States. I am confident that the number will be less than 5%.**

Please note that merely filling in "USA" as the location does not serve as definitive proof for anything, my friend.

<img width="257" alt="Screenshot 2023-07-22 at 09 37 33" src="https://github.com/filecoin-project/notary-governance/assets/70132857/4c7bf8e5-ac11-4bb5-b371-cda7e67e2259">

**Comment by c12xf** [comment]
@herrehesse  My friend,look at what you did today, it's too bad.
Your team's regional discrimination behavior is too obvious. Before I provide evidence, your team has never checked the LDN of the United States region. In your words, only fill in the LDN of the 'United States'; After I present the evidence. You have responded appropriately, such as # 944 # 943 # 945, and the LDNs for the issues raised by your team have been closed for more than 2 weeks.
I would like to emphasize once again that even if there are only 200 LDNs filled out for the 'United States', your team has never checked them before I presented the evidence, until today.

**Comment by raghavrmadya** [comment]
在中国有很多关于filecoin的群聊，包括技术群，数字货币交易群
There are many group chats about filecoin in China, including technology groups and digital currency trading groups
其中我在一个群找到了一个LDN的申请教程，我之前就是按照这个教程申请的，教程中明确说明了要激活钱包，0.01个0.1个币都可以，我没用过数字货币交易所，于是我找群里的人购买了0.01个币
Among them, I found an LDN application tutorial in a group. I previously applied according to this tutorial, which clearly stated that to activate a wallet, 0.01 or 0.1 filecoin can be used. I have not used a digital currency exchange before, so I purchased 0.03 filecoin from people in the group
据我在群里看他们的聊天记录，#914 #915的人都是看这个教程后才申请的，并且他们也明确表示是找相同的人购买的filecoin货币，所以这去证明我们是同一实体，这就是你的证据吗？
According to their Chat log I read in the group, people of #914 #915 only applied after reading this tutorial, and they also clearly stated that they bought filecoin currency from the same person, so this proves that we are the same entity. Is this your evidence?
教程PDF文档连接Tutorial PDF Document Connection ：https://smallpdf.com/cn/file#s=6e110e24-f490-4047-bdf3-5ba545c170b1
中国有很多矿工去交易所甚至个人购买

**Comment by raghavrmadya** [comment]
https://developfil.nyc3.cdn.digitaloceanspaces.com/fundingaddress.svg

**Comment by xiaoming123222** [comment]
Thanks to RG！

**Comment by c12xf** [comment]
and #914 ？
